nextflow.config

```diff
@@ -134,6 +135,20 @@ params {
     read_format = null
     umi_read = null
 
+    // -----------------------
+    // MiXCR option
+    // -----------------------
+    imgt_mixcr = null
+    kit = null
```

Comment:
```suggestion
    mixcr_preset = null
```

Reply:
Let's keep all mixcr params starting with `mixcr` and being as close as possible to the naming when directly using mixcr